modules/odc-deploying-java-applications.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// applications/application_life_cycle_management/odc-deploying-java-applications.adoc
+
+[id="odc-deploying-java-applications_{context}"]
+= Upload JAR files for easy deployment of Java applications
+
+You can use the JAR files in the *Topology* view of the *Developer* perspective to deploy your Java applications. You can upload a JAR file using the following options: 
+
+* Navigate to *+Add* view of the *Developer* perspective, click the *Upload JAR file* tile through the *From Local Machine* option. Browse and select your JAR file or drag and drop a JAR file to deploy your application.
+
+* Navigate to the *Topology* view and use the *Upload JAR file* option or drag and drop a JAR file to deploy your application.
+
+* Navigate to the Context Menu in the *Topology* view and use the *Upload JAR file* option to upload your JAR file to deploy your application.
+
+Use the following instructions to upload a JAR file in the *Topology* view to deploy a Java application.
```

Comment:
I think a colon after this sentence would help precede the procedure.

Reply:
Sure! Thanks.